process/dataAggregator.go

```diff
@@ -36,7 +37,8 @@ func (da *dataAggregator) ProcessHyperBlock(outportBlock *outport.OutportBlock)
 	hyperOutportBlock := &data.HyperOutportBlock{}
 	hyperOutportBlock.MetaOutportBlock = outportBlock
 
-	notarizedShardOutportBlocks := make([]*data.NotarizedHeaderOutportData, 0)
+	n := len(outportBlock.NotarizedHeadersHashes)
+	notarizedShardOutportBlocks := make([]*data.NotarizedHeaderOutportData, 0, n)
```

Comment:
optinal, more like a preference thing: suggestion to use len(...) directly instead of setting `n` variable